app/models/metric/ci_mixin/targets.rb

```diff
@@ -6,6 +6,7 @@ def perf_capture_always?
     when ManageIQ::Providers::Kubernetes::ContainerManager::Container then true
     when ManageIQ::Providers::Kubernetes::ContainerManager::ContainerGroup then true
     when ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode then true
+    when ContainerImage then true
```

Comment:
I'm not sure we need or want this here because it has to do with capturing metrics and I don't we are planning on capturing metrics for images. Not even sure they would be available. I think we just need to roll up metrics from child containers.

Perhaps @agrare can comment on that

Reply:
Wow this method is gross, so we're doing a `case` on the class the mixin is mixed into...instead of just defining the method in the class directly?

I don't see `perf_capture_enabled` (which is what is calling `perf_capture_always?`) being used on container resources.  It is usually used e.g. `enabled_host_ids = hosts.select(&:perf_capture_enabled?).index_by(&:id)`

I think we should drop `ContainerImage` from here, then re-evaluate how this method is used to be more pluggable (e.g. define `perf_capture_always?; true; end` in `Openstack::InfraManager::Host` class not in a core mixin.